File: csharp/DVSClient/DVSClient/server/address/format/RestApiAddressFormatMetadata.cs
using Newtonsoft.Json;

namespace DVSClient.Server.Address.Format
{
    public class RestApiAddressFormatMetadata
    {
        [JsonProperty("address_info")]
        public RestApiAddressMetadataInfo? AddressInfo { get; set; }

        [JsonProperty("barcode")]
        public RestApiAddressMetadataBarcode? Barcode { get; set; }

        [JsonProperty("route_classification")]
        public RestApiAddressMetadataRouteClassification? RouteClassification { get; set; }

        [JsonProperty("address_classification")]
        public RestApiAddressMetadataAddressClassification? AddressClassification { get; set; }

        [JsonProperty("dpv")]
        public RestApiAddressMetadataDpv? Dpv { get; set; }
    }
}

File: typescript/src/server/address/format/restApiAddressFormatMetadata.ts
import { RestApiAddressMetadataAddressClassification } from "./restApiAddressMetadataAddressClassification";
import { RestApiAddressMetadataBarcode } from "./restApiAddressMetadataBarcode";
import { RestApiAddressMetadataDpv } from "./restApiAddressMetadataDpv";
import { RestApiAddressMetadataInfo } from "./restApiAddressMetadataInfo";
import { RestApiAddressMetadataRouteClassification } from "./restApiAddressMetadataRouteClassification";

export type RestApiAddressFormatMetadata = {
    address_info?: RestApiAddressMetadataInfo;
    barcode?: RestApiAddressMetadataBarcode;
    route_classification?: RestApiAddressMetadataRouteClassification;
    address_classification?: RestApiAddressMetadataAddressClassification;
    dpv?: RestApiAddressMetadataDpv;
};